### DISCUSSION TITLE
Shortcuts in alpha dashboard do not work

### DISCUSSION THREAD

**Discussion by vendion** [nix-community/nixvim#1401] [Q&A]
I'm trying to get the alpha plugin setup on my system, but using the example for `plugins.alpha.layout' found on this page https://nix-community.github.io/nixvim/plugins/alpha/layout.html doesn't work as expected.

The "New File" and "Quit Neovim" options show up, but when I press either of the keys (`n` or `q`) nothing happens. I was wondering in the syntax for defining the shortcuts changed recently or if someone has a working example they don't mind sharing. Another suggestion would be adding a `extraConfig` to the Alpha plugin module that allows passing in pure lua like other plugins have allowing me to just use my existing Alpha config.

**Comment by siph** [comment]
`alpha` was refactored (relevant discussion: https://github.com/nix-community/nixvim/issues/944) a while ago. I've never had luck using `on_press.__raw` so I use [`opts.keymap`](https://github.com/goolord/alpha-nvim/blob/41283fb402713fc8b327e60907f74e46166f4cfd/doc/alpha.txt#L91).

I'm sure there are better ways to implement this but here is what I'm doing: https://github.com/siph/nixvim-flake/blob/7f3813d84abcdf6686b7e16bfe4a067e05e1ec0e/config/visuals/alpha.nix#L65-L102
